README.md

```diff
@@ -2,7 +2,17 @@
 
 [![npm](https://img.shields.io/npm/v/iab-vast-model.svg)](https://www.npmjs.com/package/iab-vast-model) [![Dependencies](https://img.shields.io/david/zentrick/iab-vast-model.svg)](https://david-dm.org/zentrick/iab-vast-model) [![Build Status](https://img.shields.io/circleci/project/github/zentrick/iab-vast-model/master.svg)](https://circleci.com/gh/zentrick/iab-vast-model) [![Coverage Status](https://img.shields.io/coveralls/zentrick/iab-vast-model/master.svg)](https://coveralls.io/r/zentrick/iab-vast-model) [![JavaScript Standard Style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
 
-Base classes for IAB VAST handling. Supports the full IAB VAST 3.0 specification.
+Base classes for IAB VAST handling. Supports the full IAB VAST 2.0, 3.0 and 4.0
+specifications.
+
+Note that the `<Creatives>`, `<CreativeExtensions>`, `<Extensions>` and
+`<Verifications>` are not directly recorded since they have no attributes. For
+instance, the list of `<creative>` are recorded directly by `<Inline>` and
```

Comment:
`Creative` should be capitalized. Also, `InLine` has a capital L.

I found at least one occurrence of `Inline` with a lowercase L in the JSDocs so maybe do a global find-and-replace as well.